sdk/storage/storage-file-datalake/test/filesystemclient.spec.ts

```diff
@@ -130,6 +131,25 @@ describe("DataLakeFileSystemClient", () => {
     assert.deepEqual(result.metadata, metadata);
   });
 
+  it("create with encryption scope", async function (this: Context) {
+    let encryptionScopeName;
+    try {
+      encryptionScopeName = getEncryptionScope();
+    } catch {
+      this.skip();
```

Comment:
>this.skip(); [](http://example.com/codeflow?start=6&length=12)

Only need to skip this in live mode.
And please keep track of it and adjust the pipeline env to enable this in live test when available.

Reply:
I'll send another PR to update test resource and enable these test cases in datalake and blob package.